2-api/models.md

```diff
@@ -90,22 +92,22 @@ These models are experimental because they are trained on a single source of com
 | `UNSUBSTANTIAL` | exp. | Trivial or short comments. | en | 2 |
 
 
-## Model architecture
+## Attribute architecture
 
-Most of our models are [Convolutional Neural Network](https://en.wikipedia.org/wiki/Convolutional_neural_network) (CNN) trained with [word-vector](https://www.tensorflow.org/tutorials/word2vec) inputs. You can also train your own [deep CNN for text classification](http://www.wildml.com/2015/12/implementing-a-cnn-for-text-classification-in-tensorflow/) on our [public toxicity dataset](https://figshare.com/articles/Wikipedia_Talk_Labels_Toxicity/4563973), and explore our [open-source model training tools](https://github.com/conversationai/conversationai-models) to train your own models. Our [Toxicity Kaggle Competition](https://kaggle.com/c/jigsaw-toxic-comment-classification-challenge) also has lots of useful resources to help build your own models.
+Most of our attributes are [Convolutional Neural Network](https://en.wikipedia.org/wiki/Convolutional_neural_network) (CNN) trained with [word-vector](https://www.tensorflow.org/tutorials/word2vec) inputs. You can also train your own [deep CNN for text classification](http://www.wildml.com/2015/12/implementing-a-cnn-for-text-classification-in-tensorflow/) on our [public toxicity dataset](https://figshare.com/articles/Wikipedia_Talk_Labels_Toxicity/4563973), and explore our [open-source attribute training tools](https://github.com/conversationai/conversationai-models) to train your own attributes. Our [Toxicity Kaggle Competition](https://kaggle.com/c/jigsaw-toxic-comment-classification-challenge) also has lots of useful resources to help build your own attributes.
 
-## Model Cards
+## Attribute cards
 
-For each production model, we aim to publish an associated "Model Card" that shares details about intended usage, model training processes, and evaluation results.
+For each production attribute, we aim to publish an associated "Attribute card" that shares details about intended usage, attribute training processes, and evaluation results.
```

Comment:
I think Olivia mentioned in an earlier comment that we should call these "Model cards" to be consistent with other messaging.